chain-spec.json

```diff
@@ -14,7 +14,6 @@
 		"eip161dTransition": "0x0",
 		"eip155Transition": "0x0",
 		"eip98Transition": "0x7fffffffffffff",
-		"eip86Transition": "0x7fffffffffffff",
```

Comment:
Why has EIP 86 been removed?

Reply:
https://github.com/paritytech/parity-ethereum/pull/9268

Reply:
TL;DR: will take your word for it that it is a reasonable change.